docs/content/preview/integrations/camunda.md

```diff
@@ -194,14 +170,12 @@ To deploy the process, do the following:
 
 1. Click the deploy button in Camunda Modeler, enter the deployment name "Payment Retrieval", and click the Deploy button.
 
-    \
-    If you're running Camunda Modeler version 3.0.0 or later, you need to provide the URL of an Endpoint Configuration along with the deployment details. This can be either the root endpoint to the REST API (such as <http://localhost:8080/engine-rest>) or an exact endpoint to the deployment creation method (such as <http://localhost:8080/engine-rest/deployment/create>). <br/><br/>
+    If you're running Camunda Modeler version 3.0.0 or later, you need to provide the URL of an Endpoint Configuration along with the deployment details. This can be either the root endpoint to the REST API (such as <http://localhost:8080/engine-rest>) or an exact endpoint to the deployment creation method (such as <http://localhost:8080/engine-rest/deployment/create>).
 
     ![alt text](https://docs.camunda.org/get-started/quick-start/img/modeler-deploy2.png)
```

Comment:
image doesn't display - need to add camunda.org domain

Reply:
@polarweasel added the entry in `netlify.toml`. Thanks!